src/measure_theory/function/locally_integrable.lean

```diff
@@ -78,8 +78,10 @@ begin
     exact h this }
 end
 
-section real
-variables [opens_measurable_space X] {A K : set X} {g g' : X → ℝ}
+section mul
+
+variables [opens_measurable_space X] {A K : set X} {g g' : X → R}
+[normed_ring R] [second_countable_topology R]
```

Comment:
```suggestion
[normed_ring R] [second_countable_topology_either X R]
```
It is enough to have either `X` or `R` to be second-countable. In this way, you can drop the second countability assumptions on `A` in the other file.